views/default/page/components/message.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Elgg message element
+ *
+ * @uses $vars['type']      The type of message (error, success, warning, help, notice)
+ * @uses $vars['title']     Optional title text, will default to the type
+ * @uses $vars['icon_name'] Optional iconname to override default icon
```

Comment:
`icon` would be consistent with other views

Reply:
will do

Reply:
I looked and found other icon_name $vars keys.

Reply:
hmmm icon means in a lot of cases something else than just the icon_name... think i will leave it as is for now (icon = html, icon_name is used in elgg_view_icon)